BreadCrumb Control/BreadCrumb Control/BreadCrumb.swift

```diff
@@ -170,6 +182,14 @@ public class CBreadcrumbControl: UIScrollView {
     }
 
     
+    private var buttonHeight: CGFloat {
```

Comment:
`frame.size.height` is great. I've been thinking about the `kBreadcrumbHeight` constant and I wonder if there's a usecase for someone to have a fixed height even though the `frame.size.height` is larger. Perhaps `kBreadcrumbHeight` isn't needed? What do you think?

Reply:
That's good discussion.
I think, if someone needs fixed height, maybe it needs other customize. So we don't have to provide fixed height. 

So, I'll delete `kBreadcrumbHeight` and `isButtonHeightFlexible`. How about?

Reply:
I think that's the best option!

Reply:
Does it still need to be a computed property?

Reply:
Oops, it's no longer needed.